src/core/frame/py_frame.cc

```diff
@@ -533,16 +533,20 @@ void Frame::set_nrows(const Arg& nr) {
 
 static const char* doc_shape =
 R"(
-Tuple with (nrows, ncols) dimensions of the Frame.
+Tuple with ``(nrows, ncols)`` dimensions of the frame.
+
+This property is read-only.
 
 Parameters
 ----------
 return: Tuple[int, int]
+    Tuple with two integers: the first is the number of rows, the
+    second is the number of columns.
 
 See also
 --------
-- :data:`.nrows`: getter for the number of rows of the frame;
-- :data:`.ncols`: getter for the number of columns of the frame.
+- :data:`.nrows` -- getter for the number of rows;
+- :data:`.ncols` -- getter for the number of columns.
```

Comment:
Will `--` be replaced with `—` when generating the docs?

Reply:
yes